### ISSUE TITLE
[FrontEnd] Criação de tela de cadastro Mobile

### ISSUE THREAD

**Issue by JohnnyKamigashima** [TechRate-Lab/TR-Frontend#60]
A tela de cadastro foi criada pelo time de UX.
Como usuário, quero me cadastrar no sistema,
Para poder deixar meus depoimentos.

Critérios de aceite:
- Deve seguir o design do UX
- Deve ser responsivo
- Deve possuir conexão segura com o backend de cadastro.

**Comment by luizpbello** [comment]
Já da pra puxar aqui @JohnnyKamigashima

**Comment by JohnnyKamigashima** [comment]
Afirmativo @luizpbello

**Comment by luizpbello** [comment]
@JohnnyKamigashima 
Acho que consegui fazer, só falta a integração com a API.
E também por algum motivo não está subindo para produção, só ta rodando localmente. Ainda não consegui entender porque não está funcionando na Vercel.

O video abaixo eu fiz pra mostrar como ficou:



https://github.com/TechRate-Lab/TR-Frontend/assets/88148849/3b73f4f0-bc97-4c39-b802-d971052f0bf5

**Comment by JohnnyKamigashima** [comment]
Baixei o codigo e rodei local aqui, ficou bele!
Que tal @GabiStein1  ? Podemos concluir essa issue?

![Image](https://github.com/TechRate-Lab/TR-Frontend/assets/50176581/1002a063-80e6-493e-9cc2-1e89615ba9f8)

**Comment by jeffbcampos** [comment]
@JohnnyKamigashima na tela de cadastro seria necessário o campo de senha também não? Nossa tabela recebe Nome, email e Senha

**Comment by JohnnyKamigashima** [comment]
Agora me veio a duvida, é melhor a gente pedir para definir a senha antes ou depois do email de confirmação?

**Comment by jeffbcampos** [comment]
acredito que seja isso. A confirmação de email viria em seguida, liberando o acesso a plataforma utilizando email e senha.

**Comment by luizpbello** [comment]
Agora o deploy ta funcionando.

A nova url é: https://takeapeek.vercel.app

**Comment by GabiStein1** [comment]
Concordo pessoal! Para o cadastro precisamos incluir a senha e confirmação de senha. @LaisIsfer poderiam verificar se concordam também?

**Comment by luizpbello** [comment]
Com a saída da Laís e do Francisco, e a ausencia da Emanuelle, acredito que ainda não temos novidades. A não ser que a gente implemente seguindo o padrão já criado., Só adicionando os campos mesmo.

**Comment by GabiStein1** [comment]
Podem seguir considerando o mesmo padrão, apenas adicionando os campos! @luizpbello

**Comment by luizpbello** [comment]
@GabiStein1 e @JohnnyKamigashima 

Campos adicionados:



https://github.com/TechRate-Lab/TR-Frontend/assets/88148849/fcbd6535-9af0-47af-a78b-3f585624d464

Ai @maydandrea as mensagens de erros são provisórias, assim que a gente tiver a parte visual delas prontas, a gente altera caso necessário.

**Comment by JohnnyKamigashima** [comment]
Achei  muito bom @luizpbello , so uma duvida, o texto dentro do box parece estar colado dentro da borda do box, é para ser assim mesmo @GabiStein1 ?

**Comment by GabiStein1** [comment]
Bem notado @JohnnyKamigashima . Seria legal se tivesse um espaço de um ou dois caracteres antes de começar o texto. Dá pra incluir @luizpbello ?
